### PR TITLE
Show if a PR has conflicting files on the PR lists

### PR DESCRIPTION
**Pull request by jamesorlakin**
As hinted in #8659, this PR adds a simple message to PR lists if a PR has conflicting files. This is all achieved through tweaks to templates - `ConflictingFiles` array already exists on PR objects. I've chosen mirror as the octicon but I'm sure there's a better one!

image

### CODE REVIEW

File: templates/repo/issue/list.tmpl [hunk 1]
```diff
@@ -212,16 +212,23 @@
...
-                    {{if .IsPull }}
-                        {{if (index $.CommitStatus .PullRequest.ID)}}
-                            {{template "repo/commit_status" (index $.CommitStatus .PullRequest.ID)}}
+					{{if .IsPull }}
+						{{if (index $.CommitStatus .PullRequest.ID)}}
+							{{template "repo/commit_status" (index $.CommitStatus .PullRequest.ID)}}
 						{{end}}
 					{{end}}
 
+
```
Comment (guillep2k) [inline templates/repo/issue/list.tmpl]: Extra line not required

**Review by guillep2k — APPROVED** [review]
Nice! Only one nit.

**Comment by lafriks** [comment]
It would be better to have this not in title but in line below before milestone with diff icon

**Comment by jamesorlakin** [comment]
I agree, I'll move it below

**Comment by lafriks** [comment]
Please also update screenshot when done

**Comment by jamesorlakin** [comment]
The tweaks now look like this:
image

It should always appear at the end.

**Comment by jolheiser** [comment]
Out of scope for this PR, but we might need to consider adding some separators in that area soon. 😂

**Review by jolheiser — APPROVED** [review]

**Comment by jamesorlakin** [comment]
Haha, I was debating looking at adding some logic to pop in a `|` symbol or something!

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
Btw can you please send PR to add singular/plural to text (ex 1 conflicting file). You can check activity.tmpl for sample. There is method `TrN` to do that

**Pull request merged**